hikari/commands.py

```diff
@@ -191,6 +197,12 @@ class PartialCommand(snowflakes.Unique):
     app: traits.RESTAware = attr.field(eq=False, hash=False, repr=False)
     """The client application that models may use for procedures."""
 
+    name_localizations: undefined.UndefinedOr[typing.Mapping[str, str]] = attr.field(factory=dict, kw_only=True)
+    """A set of name localizations for this command"""
+
+    description_localizations: undefined.UndefinedOr[typing.Mapping[str, str]] = attr.field(factory=dict, kw_only=True)
+    """A set of description localizations for this command"""
```

Comment:
```suggestion
    name_localizations: undefined.UndefinedOr[typing.Mapping[str, str]] = attr.field(factory=dict)
    """A set of name localizations for this command."""

    description_localizations: undefined.UndefinedOr[typing.Mapping[str, str]] = attr.field(factory=dict)
    """A set of description localizations for this command."""
```

Could you please also move them at the end of this object for consistency?

Additionally, is there a need for the default to be specified here? Not like anyone will create these objects themselves